CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.cpp

```diff
@@ -17,7 +17,10 @@ Kartaclysm::StateRacing::StateRacing()
 	m_bSuspended(true),
 	m_pPauseDelegate(nullptr),
 	m_uiNumRacers(0),
-	m_bCountdown(false)
+	m_bCountdown(false),
+	m_bCountdownToFinish(false),
```

Comment:
If we're adding a second variable with "Countdown" in its name, we should expand the first variable's name to something like "StartingCountdown" to avoid confusion